automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookDataSyncingSpec.scala

```diff
@@ -155,10 +170,10 @@ class NotebookExtensionSpec extends ClusterFixtureSpec with NotebookTestUtils {
     }
 
     "Locked by another user and playground mode transition handled correctly" in { clusterFixture =>
-      val sampleNotebook = ResourceFile("bucket-tests/gcsFile3.ipynb")
+      val sampleNotebook = ResourceFile("bucket-tests/gcsFile4.ipynb")
```

Comment:
why 2 3 and 4 now?

Reply:
I observed the edit mode and playground mode tests were using the same file (`gcsFile.ipynb`) and the same cluster. I thought they may be stepping on each other. I made each test use a different notebook to be independent.